src/sentry/models/projectownership.py

```diff
@@ -31,6 +32,18 @@
 _Everyone = enum.Enum("_Everyone", "EVERYONE")
 
 
+# JSON object representing this model in API response
+class ProjectOwnershipResponse(TypedDict):
```

Comment:
should this go in the serializer file instead of the model file?

Reply:
This is actually not needed at all. I can directly return the serializer 🤦‍♀️ Removing